src/Rector/Dynamic/ClassReplacerRector.php

```diff
@@ -56,7 +56,9 @@ private function resolveNewNameFromNode(Node $node): string
     {
         $nameNode = $this->resolveNameNodeFromNode($node);
 
-        return $this->oldToNewClasses[$nameNode->toString()];
+        if ($nameNode !== null) {
+            return $this->oldToNewClasses[$nameNode->toString()];
+        }
```

Comment:
This should be probably covered by tests too. PHPStan doesn't fix that

Reply:
Yes, I thought about that too. Let's stay with PHPStan fixes in this PR, and expand coverage in the next PR?